spf4j/spf4j-core-app/pom.xml

```diff
@@ -39,16 +39,16 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.0</version>
+                <version>${compiler.plugin.version}</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>${source.version}</source>
```

Comment:
can we use the java.version property from the main pom here, and not define a new one?

Reply:
I have created a new PR #8642 with the changes you asked for.